From: Ben Hutchings <ben@decadent.org.uk>
Date: Wed, 14 Feb 2018 19:41:40 +0000
Subject: SCSI: Avoid ABI change in 3.2.97
Forwarded: not-needed

Commit 28a0bc4120d3 ("scsi: sd: Implement blacklist option for WRITE
SAME w/ UNMAP") added a new bitfield to struct scsi_device.  This
takes up one padding bit, so it's not a real ABI change and we can
just hide it from genksyms.

---
--- a/include/scsi/scsi_device.h
+++ b/include/scsi/scsi_device.h
@@ -152,7 +152,10 @@ struct scsi_device {
 	unsigned no_read_capacity_16:1; /* Avoid READ_CAPACITY_16 cmds */
 	unsigned is_visible:1;	/* is the device visible in sysfs */
 	unsigned broken_fua:1;		/* Don't set FUA bit */
+#ifndef __GENKSYMS__
 	unsigned unmap_limit_for_ws:1;	/* Use the UNMAP limit for WRITE SAME */
+	/* 27 bits padding */
+#endif
 
 	DECLARE_BITMAP(supported_events, SDEV_EVT_MAXBITS); /* supported events */
 	struct list_head event_list;	/* asserted events */
